openy_af4_vue_app/src/App.vue

```diff
@@ -433,6 +473,9 @@ export default {
         selectedWeeks: [],
         selectedLocations: [],
         selectedActivities: [],
+        selectedInMemberships: 0,
```

Comment:
Could it be Bool instead of Int ?

Reply:
@rollki, but we store a value from this field in an integer, that's why I used an integer too

Reply:
Then for now, we'll leave it as it is.

Reply:
Okkk, I see u changed to bool, thanks!